snippets/terms.md

```diff
@@ -1,6 +1,6 @@
 ## Conditions of OpenSAFELY data use
 * OpenSAFELY operates as a trusted research platform; no record level data is permitted to be extracted from the platform.
-* The data held in the OpenSAFELY-EMIS/-TPP NHS England COVID-19 research platform can only be analysed for [COVID-19 related purposes](https://www.england.nhs.uk/contact-us/privacy-notice/how-we-use-your-information/covid-19-response/coronavirus-covid-19-research-platform/).
+* The data held in the NHS England OpenSAFELY COVID-19 service can only be analysed for [COVID-19 related purposes](https://www.england.nhs.uk/contact-us/privacy-notice/how-we-use-your-information/covid-19-response/coronavirus-covid-19-research-platform/).
```

Comment:
This link doesn't exist.

Reply:
This is the link that will replace the one that has now expired- https://digital.nhs.uk/coronavirus/coronavirus-covid-19-response-information-governance-hub/the-nhs-england-opensafely-covid-19-service-privacy-notice

Thanks for checking this

Reply:
You've not committed the new link, @LiamHart-hub; just the old link. I'll open a PR now.

Reply:
My apologies. 

I will rectify this now.